docker/ray/Dockerfile

```diff
@@ -1,11 +1,23 @@
 ARG GPU
-FROM rayproject/ray-deps:nightly"$GPU"
-ARG WHEEL_PATH
-ARG FIND_LINKS_PATH=".whl"
+ARG DOCKER_PREFIX
+
+# Pull the wheel from the ray-wheel build
+FROM "$DOCKER_PREFIX"ray-wheel:nightly"$GPU" AS raywheel
+WORKDIR /ray
```

Comment:
What is the reason for switching from COPY -> from another image?

Reply:
So we copy from another image so that we get the right wheel since the raywheel image contains the wheel built for our arch.